test/functional/mempool_packages.py

```diff
@@ -58,10 +57,17 @@ def run_test(self):
 
         # Mine some blocks and have them mature.
         self.nodes[0].generate(101)
-        utxo = self.nodes[0].listunspent(10)
-        txid = utxo[0]['txid']
-        vout = utxo[0]['vout']
-        value = utxo[0]['amount']
+        utxos = []
+        for utxo in self.nodes[0].listunspent(10):
+            # Skip change/fees we scooped up
+            if utxo['amount'] != Decimal(50):
+                continue
+            utxos.append(utxo)
+        assert len(utxos) != 0
```

Comment:
Since you're using 2 of the utxos, should you assert it's at least length 2?

Reply:
i was lazy and didn't want to look up the greater than assert... fixed